source/taup/index.rst

```diff
@@ -0,0 +1,7 @@
+TauP
+====
+
+.. toctree::
+
+   intro
```

Comment:
introduction or intro?

Reply:
let's use a word instead of shortcut?

Reply:
Yes to me

Reply:
okay. make it as a rule in #27 ?

Reply:
OK. you can directly push to that branch.

Reply:
Done.